apps/web/package.json

```diff
@@ -12,7 +12,7 @@
     "jest": "jest src",
     "types": "sapper build && tsc && svelte-check --treshold error",
     "installCourse": "./scripts/installExternalCourse.sh",
-    "installAllExternalCourses": "./scripts/installExternalCourses.sh",
+    "installAllExternalCourses": "python ./scripts/installExternalCourses.py ../../config/courses.json",
```

Comment:
```suggestion
    "installAllExternalCourses": "poetry python ./scripts/installExternalCourses.py ../../config/courses.json",
```

Reply:
btw if we are all the way going to re-writing scripts in Python (which would probably make sense, if we want to support (simple) Windows setups) then poetry scripts could be used for this perhaps, instead of yarn :thinking:

Reply:
or else I'm also considering using a Makefile, or similar solution to have a "central" script runner in non-hacky ways. At least using `yarn` to run Python scripts is something some Python developers I know would take issue with :sweat_smile: What do you think?

Reply:
Makefile would require some kind of a make command on your computer :) Not a very Windowsy thing.

Reply:
The current script only uses standard-Python modules so there is not a lot of need for poetry. For now.

Reply:
basically poetry scripts are just python commands that are added to path. for sure that's a lot less flexible than what yarn does

Reply:
> Makefile would require some kind of a make command on your computer :) Not a very Windowsy thing.

that's a good point